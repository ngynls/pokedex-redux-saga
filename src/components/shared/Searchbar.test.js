import React from 'react';
import renderer from 'react-test-renderer';
import SearchBar from './Searchbar'

test('matches snapshot of SearchBar component', ()=>{
    const tree=renderer.create(<SearchBar />).toJSON();
    expect(tree).toMatchSnapshot();
});