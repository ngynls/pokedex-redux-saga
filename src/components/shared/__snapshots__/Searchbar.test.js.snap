// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot of SearchBar component 1`] = `
<div
  className="search-filter-bar mt-4"
>
  <input
    aria-label="Search"
    className="form-control"
    placeholder="Search for a Pokemon"
    type="search"
  />
</div>
`;
